### PR TITLE
[pointer_interceptor] Update min Flutter version

### PR DESCRIPTION
**Pull request by ditman**
The change in 0.9.2 requires Flutter 2.10, but the min SDK version
wasn't updated so it would break people who are still on 2.8 (or
earlier).

0.9.2 has been retracted on pub.dev to address that; this will
effectively re-release 0.9.2 with the right constraints as 0.9.3.

_Similar to https://github.com/flutter/plugins/pull/4735_

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

File: packages/pointer_interceptor/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
-## 0.9.2
+## 0.9.3
 
 * Marked `PointerInterceptor` as invisible, so it can be optimized by the engine.
+* Require minimal version of flutter SDK to be `2.10`
+
+## 0.9.2
```
Comment (stuartmorgan) [inline packages/pointer_interceptor/CHANGELOG.md:6]: Looks like you added trailing whitespace here.

File: packages/pointer_interceptor/pubspec.yaml [hunk 2]
```diff
@@ -2,11 +2,11 @@ name: pointer_interceptor
 description: A widget to prevent clicks from being swallowed by underlying HtmlElementViews on the web.
 repository: https://github.com/flutter/packages/tree/main/packages/pointer_interceptor
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+pointer_interceptor%22
-version: 0.9.2
+version: 0.9.3
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.17.0"
+  flutter: ">=2.10.0-0"
```
Comment (stuartmorgan) [inline packages/pointer_interceptor/pubspec.yaml:9]: Per offline discussion, I'd much rather we hold off to see if we can get master bumped to 2.11 today and then use a release version here.

File: packages/pointer_interceptor/CHANGELOG.md (hunk 1 above)
Comment (ditman) [reply packages/pointer_interceptor/CHANGELOG.md:6]: I looked for it everywhere, but couldn't find it :P

File: packages/pointer_interceptor/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
-## 0.9.2
+## 0.9.3
 
 * Marked `PointerInterceptor` as invisible, so it can be optimized by the engine.
```
Comment (stuartmorgan) [inline packages/pointer_interceptor/CHANGELOG.md:3]: This shouldn't be moved from 0.9.2 to 0.9.3. It was in 0.9.2, and there are ways to get 0.9.2 (force-pinning, using old versions of Flutter).

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/pointer_interceptor/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,11 @@
+## 0.9.3
+
+* Require minimal version of flutter SDK to be `2.10`
+
 ## 0.9.2
 
 * Marked `PointerInterceptor` as invisible, so it can be optimized by the engine.
+* (Version Retracted. This attempted to use an API from flutter `2.10` in earlier versions of flutter. Fixed in v0.9.3)
```
Comment (stuartmorgan) [inline packages/pointer_interceptor/CHANGELOG.md:8]: Nit: "Flutter" rather than "flutter" in both instances.

**Review by stuartmorgan — APPROVED** [review]
LGTM with nit

**Pull request merged**